help/pwsafeRU/html/credits.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-<title>Благодарности</title><!-- pwsafe version: 3.12--><!-- Modified: 10/12/2007 --><!--Popup control -->
+    <meta http-equiv="content-type" content="text/html; charset=Windows-1251">
+<title>�������������</title><!-- pwsafe version: 3.12--><!-- Modified: 10/12/2007 --><!--Popup control -->
```

Comment:
This file renders weird unlike the others

Reply:
It should be fine. It seems, that Github can't detect codepage changes in these files (it was surprise to me, that in other files it recoded diff lines for new revision in preview). You could take a look at raw file content at my repo, for example https://raw.githubusercontent.com/pm-kan/pwsafe/ru-help-encoding/help/pwsafeRU/html/credits.html (don't forget to change browser's encoding to Windows-1251)

Reply:
Yeah, that worked. Not sure why the others render fine by default.